src/views/index-view.js

```diff
@@ -0,0 +1,31 @@
+export default Backbone.View.extend({
+  tagname: 'ul',
+
+  attributes: {
+    class: 'index-list',
+  },
+
+  initialize() {
+    this.render();
+
+    this.listenTo(this.collection, 'sync', this.render);
+  },
+
+  render() {
+    this.$el.html(this.template(this.collection));
+  },
+
+  template(collection) {
+    return collection.map((model) => {
+      return `<ul class="run-list">
+        <li class="run-log">
+          <span class="list-items date">${model.get('date')}</span>
+          <span class="list-items time">${model.get('time')}</span>
+        </li>
+        <li>
```

Comment:
Since this `li` is in your `map` function in the collection, you are creating a "New" button for each item in the collection.


Instead try appending only one new `li` and link in the `render` function AFTER setting the `html` using the template.
This means that the starting list will be all of the items, and then you just add a list item with the add link.